tools/explicit_ci_jobs.py

```diff
@@ -0,0 +1,134 @@
+import yaml
```

Comment:
Could you add a shebang and also make this file executable?
```suggestion
#!/usr/bin/env python3
import yaml
```